ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/PackageConfigurationFactory.kt

```diff
@@ -60,25 +176,47 @@ internal object PackageConfigurationFactory {
             )
         }
 
-        val firstPackage = packageInfos.first()
-        val defaultPackage = packageInfos.firstOrNull { it.rcPackage.identifier == default } ?: firstPackage
+        return Pair(locale, packageInfos)
```

Comment:
Same tiny suggestion here about using the `to` notation, but feel free to ignore

```suggestion
        return locale to packageInfos
```